doc/source/whatsnew/v2.2.0.rst

```diff
@@ -232,6 +232,28 @@ For example:
 
     pd.date_range('2020-01-01', periods=3, freq='ME')
 
+Deprecate alias ``Q`` in favour of ``QE`` for offsets
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
This looks very similar to the previous section, perhaps we can combine them?

Reply:
yes, I agree. I combined these two sections together.